### PR TITLE
Feat/p2022 827/create smoke test for android patronage app

### PR DESCRIPTION
**Pull request by KlaudiaLapcionek**

### CODE REVIEW

**Review by MarekPawlowskiQA — COMMENTED** [review]
Also please remove app-release.apk and add currently hardcoded app to repository. 
(Remember that this is bad practice, but for now framework is on POC state :) so we can do this )

File: src/test/java/com/intive/template/steps/HomeSteps.java [hunk 1]
```diff
@@ -3,25 +3,19 @@
...
+    public void retroBoardIsOpen(){
+        Assert.assertTrue("Application is not opened", homePage.isApplicationOpen());
     }
 
-    @Then("Todo screen appear")
-    public void todoScreenAppear() {
-        Assert.assertTrue("Todo screen was not visible", homePage.isAddTodoScreenIsVisible());
+    @Then("{string} text should be visible")
+    public void emailIsVisible(String textMessage) {
+        Assert.assertEquals(homePage.getEmail(), textMessage);
```
Comment (MarekPawlowskiQA) [inline src/test/java/com/intive/template/steps/HomeSteps.java:19]: Here you can also add, description what will be print out if the assertion will fail

**Review by MarekPawlowskiQA — APPROVED** [review]
lgtm :) nice job mobile is not easy to setup

**Review by ospatrycja — APPROVED** [review]
lgtm :)

**Pull request merged**